rfcs/0006-working-in-the-open.md

```diff
@@ -0,0 +1,106 @@
+---
+created-on: 2021-03-31
+freshness: 2021-03-31
+owner: @dj2
+status: proposal
+tags:
+  - opensource
+---
+
+[La version française suit.](#résumé)
+
+# Working in the open
+## Summary
+CDS as an organization has a commitment to working in the open, inspired by our
+peers in the UK, United States, and around the world. As we develop software
+products, coding in the open provides a number of important benefits:
+ * It encourages good practice
+ * It makes collaboration easier
+ * External users can help make it better
+ * Others can learn from our work
+ * It makes it easier to share standards
+ * It improves transparency on government’s work
+ * It clarifies ownership
+ * It helps make government technology seamless
+ * It’s easier to start code in the open than to open a closed repository
```

Comment:
WDYT about saying that it saves us money as well? Just for CI for example, we are able to leverage [GitHub Actions](https://github.com/features/actions) for free because our code is public. I don't have numbers, maybe check with GitHub admins like @wmoussa-cds or @maxneuvians but I wouldn't be surprised to see that we're saving tens of thousands of dollars per month.

Reply:
Would add that this is also a selling point to attract people, can be a huge benefits for devs to be able to share things they worked on

Reply:
How about something like:
 * Releasing as open source allows us to leverage tools for open source projects, reducing overall costs
 * Allows developers to easily talk about what they're working on without issues of confidentiality.

?  Not really sure of a good way to phrase them.

Reply:
Those 2 suggestions sound good! Feel free to put them in if you're convinced

Reply:
done

Reply:
I did a presentation on this a few years back for Open First in the GoC I have some other things we might want to add in there: 

https://canada-ca.github.io/ofd-joep/assets/pdf/Open%20Source%20Code%20Calvin%20Rodo.pdf